Xamarin.Forms.Core/Cells/Cell.cs

```diff
@@ -257,5 +252,39 @@ void OnParentPropertyChanging(object sender, PropertyChangingEventArgs e)
 			if (e.PropertyName == "RowHeight")
 				OnPropertyChanging("RenderHeight");
 		}
+
+
+		#region internal element configuration
+		// This creates an internal class to keep track of IElementConfiguration because adding 
+		// IElementConfiguration<T> to the Cell itself kills performance on UWP
+		// Issue has been logged with UWP
+		public IPlatformElementConfiguration<T, Cell> On<T>() where T : IConfigPlatform
```

Comment:
see, the region says `internal element configuration`, and this is 1/public, and 2/platform configuration. regions are out of date more quickly than code comment does

Reply:
Yea I should use the word nested or inner not internal